webtau-core/src/main/java/org/testingisdocumenting/webtau/Matchers.java

```diff
@@ -89,6 +108,54 @@ public static ContainMatcher containing(Object expected) {
         return new ContainMatcher(expected);
     }
 
+    /**
+     * Contain all matcher
+     * <pre>
+     * actual(collection).should(containAll(list));
+     * </pre>
+     * @param expected collection of values to be contained in collection
+     * @return matcher instance
+     */
+    public static ContainAllMatcher containAll(Collection<Object> expected) {
+        return new ContainAllMatcher(expected);
+    }
+
+    /**
+     * Contain all matcher
+     * <pre>
+     * actual(collection).should(containAll(2, 3, "a"));
+     * </pre>
+     * @param expected var arg of expected values
+     * @return matcher instance
+     */
+    public static ContainAllMatcher containAll(Object... expected) {
+        return new ContainAllMatcher(Arrays.asList(expected));
+    }
+
+    /**
+     * Containing all matcher. Alias to containAll
+     * <pre>
+     * actual(collection).should(containAll(list));
```

Comment:
more like `myArray.should == [a, b, containingAll(list)]` , here perhaps translated to java

Reply:
this makes sense, thank you